SU-SWS/stanford_profile_helper
# D8CORE-2764: menu underline fix

# READY FOR REVIEW
- (Edit the above to reflect status)

# Summary
- Fix to the navigation menu styles

# Needed By (Date)
- Soon

# Urgency
- for v1.5.1 release

# Steps to Test

1. Check out this branch,
2. Clear cache
3. Verify that the underlines on navigation only display on hover

![Screen Shot 2020-11-30 at 8 46 36 PM](https://user-images.githubusercontent.com/284440/100698455-549ce980-334d-11eb-9fe0-abdc223b45c2.png)

Default
![Screen Shot 2020-11-30 at 8 51 56 PM](https://user-images.githubusercontent.com/284440/100698746-118f4600-334e-11eb-854b-6f8aceb4ac71.png)

Hover over external link
![Screen Shot 2020-12-02 at 12 24 19 PM](https://user-images.githubusercontent.com/284440/100927285-89b65280-3499-11eb-833e-1c4e3f9a98f4.png)




# Affected Projects or Products
D8Core

# Associated Issues and/or People
* JIRA ticket: 
** https://stanfordits.atlassian.net/browse/D8CORE-2764
** https://stanfordits.atlassian.net/browse/D8CORE-3147
** https://stanfordits.atlassian.net/browse/D8CORE-3146


- Other PRs: https://github.com/SU-SWS/stanford_profile_helper/pull/51
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)

I've put all the styles for the external links and menu links in the one general file. I'm wondering if we should break them apart?

Also, Ishi is considering that she'd like to see at least some of these styles go into Decanter. 

- Anyone who should be notified? (`@mention` them here)
@ishahism 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
